src/emitters/emitter.sqns.ts

```diff
@@ -821,7 +821,9 @@ export class SqnsEmitter implements IEmitter {
     options?: ProcessMessageOptions
   ): Promise<IFailedConsumerMessages> {
     const results = await Promise.allSettled(
-      messages.map((message) => this.processMessage(message))
+      messages.map((message) => this.processMessage(message, {
```

Comment:
no need in fifo?

Reply:
full options are already being passed in fifo